history/characters/1_azerothian.txt

```diff
@@ -12373,4 +12373,78 @@
 		add_trait=class_necromancer_6 
 	}
 	608.3.19={ death=yes }
+}
+
+#dynasty=409
+#VanCleef
+5811={
+	name=Edwin
+	dynasty=409
+	dna="digcgmdbaja"
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=6 stewardship=4 intrigue=7 learning=5
+	add_trait=intricate_webweaver add_trait=cynical add_trait=greedy add_trait=just add_trait=patient
+	disallow_random_traits = yes
+	561.8.9={ birth=yes  add_trait=creature_human }
+	592.1.15={ 
+		add_trait=class_rogue_6
+		religion=cult_of_wealth
+		add_trait=peasant_leader
+		add_claim=k_stormwind
+		effect = {
+			# Well-balanced troops
+			spawn_colonial_troops_effect = yes
+			# spawn_colonial_troops_effect = yes
+		}
+	}
+	608.4.11={ death=yes }
+}
+5812={
+	name=Vanessa
+	dynasty=409
+	female=yes
+	father=5811
+	dna="lmndmiboogc"
+	culture=azerothian religion=cult_of_wealth
+	martial=7 diplomacy=4 stewardship=4 intrigue=7 learning=5
+	595.2.11={ birth=yes  add_trait=creature_human }
+	620.9.13={ death=yes }
+}
+
+#dynasty=410
+#Thredd
+5813={
+	name=Bazil
+	dynasty=410
+	dna="kkklfgeomch"
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=6 stewardship=4 intrigue=7 learning=5
+	add_trait=intricate_webweaver
+	565.8.9={ birth=yes  add_trait=creature_human }
+	592.1.15={ 
+		add_trait=class_rogue_4
+		religion=cult_of_wealth
+		add_trait=peasant_leader
+	}
+	598.3.17={
+		employer=5811
+	}
+	608.6.15={ death=yes }
+}
+
+#dynasty=412
+5815={
+	name=Klaven
+	dynasty=412
+	dna="mortusklavn"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=5 stewardship=5 intrigue=6 learning=7
+	add_trait=scholarly_theologian
+	567.1.15={ birth=yes  add_trait=creature_human }
```

Comment:
> Klaven Mortwake is a human member of the Defias Brotherhood found in his tower in the Dust Plains, in the human territory of Westfall. He is very unstable mentally, as evidenced by his journal.

Maybe we should give him lunatic trait?

Reply:
Done in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/42ef4e94ffc962b44dd07a92441bf535e53a64a3.